docs/containerd-env-setup.md

```diff
@@ -123,6 +123,35 @@ Then restart containerd, e.g.:
 sudo systemctl restart containerd
 ```
 
+### Using Runtime-level Snapshotter in Container for Nydus
+Containerd (version >= v1.7.0) has supported developers configuring ```runtime-level snapshotter```. Following these two steps, we can use different snapshotter for different runtime.
```

Comment:
How about: Containerd (>= v1.7.0) supports configuring the `runtime-level` snapshotter. By following the steps below, we can declare runtimes that use different snapshotters: